### PR TITLE
Document deployment flags on UI and signer's Makefiles

### PR DESCRIPTION
**Pull request by italo-sampaio**
Added documentation for the values used for the `appFlags` argument on both UI and signer Makefiles

### CODE REVIEW

File: ledger/src/signer/Makefile [hunk 1]
```diff
@@ -45,7 +45,9 @@ include $(BOLOS_SDK)/Makefile.defines
 # Main app configuration
 APPNAME = "RSK Sign"
 APPVERSION = 3
-APP_LOAD_PARAMS = --appFlags 0x00 $(COMMON_LOAD_PARAMS)
+# No flags
+APPFLAGS = 0x00
+APP_LOAD_PARAMS = --appFlags $(APPFLAGS) $(COMMON_LOAD_PARAMS)
```
Comment (amendelzon) [inline ledger/src/signer/Makefile:50]: Are we actually using `APP_LOAD_PARAMS`? Maybe we could remove it if we're not.

**Review by amendelzon — APPROVED** [review]
LGTM! One minor suggestion.

File: ledger/src/signer/Makefile (hunk 1 above)
Comment (italo-sampaio) [reply ledger/src/signer/Makefile]: Nice catch! removed!

**Review by amendelzon — APPROVED** [review]
LGTM!

**Pull request merged**